src/main/java/org/opentripplanner/middleware/otp/core/util/IntUtils.java

```diff
@@ -0,0 +1,46 @@
+package org.opentripplanner.middleware.otp.core.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class IntUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(IntUtils.class);
+
+    public static int getIntFromString(String input) {
```

Comment:
Methods in this class should have javadoc.

Reply:
Will update if we are keeping OTP classes

Reply:
We are not keeping OTP classes. This class no longer exists.